stdlib/src/builtin/int.mojo

```diff
@@ -1212,6 +1217,65 @@ struct Int(
 
         writer.write(self)
 
+    @staticmethod
+    fn from_bytes[
```

Comment:
I don't think my comment before was ever properly addressed. I think this method should be on `SIMD`, not `Int`. That way the `DType` makes more sense. We can think more about what it would look like on `Int` later, and doing `int(UInt64.from_bytes(span))` doesn't seem like too much of a hit to me.

Reply:
Sorry, I missed it. Done:
https://github.com/msaelices/mojo/commit/776b4dfd36427d90eeeba05ebf69cbe4bd7d4c0e
https://github.com/msaelices/mojo/commit/265b6481a6af040058f3c3da76ed7c17cf4ae5c5

Reply:
Sorry for all the change requests, but to be clear I am comfortable merging these changes on `SIMD`, but not on `Int`. I think the `SIMD` API makes a ton of sense, but the `int` API would need some more thought since taking a `DType` feels wrong.

Reply:
Yes, actually the whole point of this PR was to help Python developers who need to migrate Python code using [int.from_bytes()](https://docs.python.org/3/library/stdtypes.html#int.from_bytes) and [int.to_bytes()](https://docs.python.org/3/library/stdtypes.html#int.to_bytes). So, to me, it's easier if we have it on the `Int` struct too, and iterate it.

Reply:
I disagree with having it on the Int struct too because we would either:

* Break code users relied on
* Never get around to revising it and have a substandard API

I'm also not compelled by the idea that the goal is to mirror a Python API considering your earlier argument in favor of breaking away from the Python API using a parameter. As I mentioned before, I think `int(UInt64.from_bytes(my_span))` is fine for now